pytensor/graph/op.py

```diff
@@ -289,7 +289,14 @@ def __call__(self, *inputs: Any, **kwargs) -> Variable | list[Variable]:
 
         """
         return_list = kwargs.pop("return_list", False)
+        name = kwargs.pop("name", None)
```

Comment:
Instead of popping we can make it an explicit optional kwarg in the call signature (so it's actually discoverable). Same for `return_list`. No idea why they went for this implicit approach